planetarium/NineChronicles
# UI/combination result

### Description

1. Use `CombinationResult` to test.

### How to test

1. Receive combination mail.

// Only in Unity Editor
1. Go to Main lobby.
2. Input `Ctrl` + `C`
3. You can see `CombinationResult` of random item which in your inventory.
4. If `Equipment` or `Consumable` are empty, Step 2 can be failed.

### Related Links

1. [(프로그래밍) 제작&결과화면 화면 UI](https://app.asana.com/0/958521740385861/1200521701229618/f)
